backend/src/main/kotlin/app/ehrenamtskarte/backend/migration/database/Schema.kt

```diff
@@ -0,0 +1,24 @@
+package app.ehrenamtskarte.backend.migration.database
+
+import org.jetbrains.exposed.dao.IntEntity
+import org.jetbrains.exposed.dao.IntEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.sql.javatime.CurrentTimestamp
+import org.jetbrains.exposed.sql.javatime.timestamp
+
+object Migrations : IdTable<Int>() {
+    override val id = integer("version").entityId()
+    override val primaryKey = PrimaryKey(id)
+
+    val name = varchar("name", length = 400).index()
```

Comment:
Not sure, if we need an index on name.
Do we need a uniqueness constraint? 

```suggestion
    val name = varchar("name", length = 400).index()
```

Reply:
Probably not on the name. And I guess the index could also be removed.